### ISSUE TITLE
link to 5" LCD not longer resolving

### ISSUE THREAD

**Issue by lwiechec** [penk/MainboardTerminal#3]
Hi,

first of all - thank you for working on this project; it is very very cool!

I managed to get the case printed - now it's time to get the rest of the elements :) I noticed that the link to the 5" round LCD does not longer resolve.
[Quick search on AliBaba](https://www.alibaba.com/trade/search?fsb=y&IndexArea=product_en&CatId=&tab=all&SearchText=5%22+round+lcd+display) gives me a couple of options; which would be fitting, in your opinion?...

Greetings,

**Comment by larsbrinkhoff** [comment]
I just tried it.  Still not resolving.  I was just about to collect everything for starting a build. :-(

**Comment by lwiechec** [comment]
yes, so it seems. I took a chance and ordered [this 5" display"](https://nl.aliexpress.com/item/4000264561778.html?gatewayAdapt=glo2nld) and it fits - it works with Raspberry Pi which I am planning to put inside until I can afford the 'proper' mainboard

**Comment by penk** [comment]
Hi @lwiechec and @larsbrinkhoff, thanks for the issue & update! Somehow I completely missed this issue... yes I believe the links you find would work, as long as there's a HDMI converter board it'll be fine.
